README.md

```diff
@@ -1,79 +1,201 @@
-# research-paper-graph
-
-## Project Setup
-If you are using IntelliJ/PyCharm Professional, you should simply be able to open the project.
-You will find some useful run configurations as well.
-
-### Backend
-The backend is a Python application with Flask framework.
-
-Some good starting resources are:
+# Research Paper Analyser
```

Comment:
I would rather call it "Research Paper Graph". We are not really analyzing papers